src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprobationestateapi/controller/ProbationDeliveryUnitController.kt

```diff
@@ -18,14 +20,24 @@ class ProbationDeliveryUnitController(
   private val getProbationDeliveryUnitService: GetProbationDeliveryUnitService
 ) {
 
-  @Operation(summary = "Retrieve Probation Delivery Unit by Code")
+  @Operation(summary = "Retrieve Probation Delivery Unit teams")
   @ApiResponses(
     value = [
       ApiResponse(responseCode = "200", description = "OK"),
       ApiResponse(responseCode = "404", description = "Result Not Found")
     ]
   )
   @GetMapping("/probationDeliveryUnit/{code}/teams")
-  suspend fun getProbationDeliveryUnitByCode(@PathVariable(required = true) code: String): Flow<TeamOverview> =
+  suspend fun getProbationDeliveryUnitTeams(@PathVariable(required = true) code: String): Flow<TeamOverview> =
     getProbationDeliveryUnitService.findTeamsByCode(code)
+
+  @Operation(summary = "Get Probation Delivery Unit by code")
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "200", description = "OK")
+    ]
```

Comment:
it does also presumably return a 404 thanks to the EntityNotFoundException thrown on line 42